server/src/logic.js

```diff
@@ -311,7 +311,13 @@ export const groupHandler = {
         if (!group) {
           return Promise.reject(Error('Invalid group!'));
         }
-        return group.addUser(user).then(() => group);
+        return user.getGroups({ where: { id: groupId } }).then((existing) => {
+          console.log(existing);
```

Comment:
remove

Reply:
aaaw. i had even fixed that and forgot to save before pushing